documentation/get-started/settings.en.md

```diff
@@ -0,0 +1,129 @@
+---
+title: General settings
+tx_slug: documentation_get-started_settings
+---
+
+# General settings
+
+In QField, you have some general settings; here you will find further details about them.
```

Comment:
If possible, I would avoid `;` in the first paragraph on a page. It breaks the reading flow.

Reply:
An intro like "the general settings page allows you to control basic appearance and behavior of the app" adds a tiny bit more context.